### PR TITLE
Add workflows to respec doc

### PR DESCRIPTION
**Pull request by BenjaminMoe**
This pull request creates a workflow directory and adds a simple example for testing. 

image

### CODE REVIEW

File: docs/sections/workflows.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+    <section>
+      <h2>Workflows</h2>
+      
+      <h3>Business Card Workflow</h3>
+      <image style='border: 1px solid #9a9a9a; border-radius: 3px; background-color: #2a303c;' src='resources/wf_businesscard.svg'/>
+      <p>This is a simple as possible workflow that describes an interaction between  two individuals. Alice and Bob are two individuals who support sending  presentations using the Traceability API. Alice issues a credential and  sends it to Bob in a Traceable Presentation.</p>
+      <b>Credentials Used:</b>
+      <ol>
+        <li><a href="undefined">Verifiable Business Card</a></li>
```
Comment (OR13) [inline docs/sections/workflows.html:10]: why undefined?

File: packages/traceability-schemas/scripts/schemas-to-vocab.js [hunk 2]
```diff
@@ -129,9 +121,49 @@ const separateSchemas = (schemaList) => {
   return { credentials, vocab };
 };
 
+const buildWorkflowSection = () => {
+  const images = {
+    "Business Card Workflow": "resources/wf_businesscard.svg"
```
Comment (OR13) [inline packages/traceability-schemas/scripts/schemas-to-vocab.js:126]: why use mermaid, if you we don't render it?

https://mermaid.js.org/config/usage.html

^ lets do this, for the `mermaid` block... and maybe add an example that uses an image, so we cover both cases.

**Review by nissimsan — APPROVED** [review]

File: packages/traceability-schemas/scripts/schemas-to-vocab.js (hunk 2 above)
Comment (nissimsan) [inline packages/traceability-schemas/scripts/schemas-to-vocab.js:126]: Having to add the image reference here seems wrong. The workflow yml should be self-contained. 

How about adding the SVG as an element alongside mermaid. 

```
mermaid: >-
  sequenceDiagram
      participant A as Alice
      ...
svg: >-
  <svg aria-roledescription="sequence"
  ...
    
```

**Review by nissimsan — APPROVED** [review]

File: docs/sections/workflows.html (hunk 1 above)
Comment (BenjaminMoe) [reply docs/sections/workflows.html:10]: i changed from `url` to `id`, I can update this.

**Review by brownoxford — APPROVED** [review]

**Pull request merged**